docs/howto/specify-splits.md

```diff
@@ -0,0 +1,48 @@
+
+The following artificial example defines the split(s) based on a CSV column:
+
+TODO: example snippet, also with `wd:` context declaration.
+
+Often, data from different splits come packaged into different files, which enable dataset consumers to only download the needed data. In such cases, the split definition is still done at the `RecordSet` level, but can refer to data file names or path.
+
+The following example is extracted from the [COCO2014 dataset Croissant config](https://github.com/mlcommons/datasets_format/blob/main/datasets/coco2014/metadata.json):
+
+<https://github.com/mlcommons/datasets_format/blob/2e05069f02a93e6154137aa267e69bd1271a95b0/datasets/coco2014/metadata.json#L173-L187>
+
+As one would expect, tools working with the Croissant config format can still infer the packaged data files needed for each split, and if a user requests loading only the "test" split, then tools can avoid downloading the "train" split files.
+
+While splits can already be used as it, a dataset publisher can add semantic information about the split names themselves, so they are easier to use in a standardized way, or can be used automatically by tools. This is the purpose of the `references` field in the above example, and this is described in the next section.
+
+## Associating a semantic meaning to split names
+
+Different names might refer to the same conceptual split. For example, `validation`, `valid`, `val`, `dev` all refer to the same split concept, and might be used indifferently depending on the dataset. The Croissant config does not intend to force dataset publishers in using a single term, but provides a way to associate a semantic meaning (ie: URL) to the used split names.
+
+The above example shows how a split information can be attached to a `RecordSet`. To associate semantic meaning to the split values, one needs to "refer" to another `RecordSet` describing those splits. In the above example, this reference is done with the `references` field:
+
+https://github.com/mlcommons/datasets_format/blob/2e05069f02a93e6154137aa267e69bd1271a95b0/datasets/coco2014/metadata.json#LL186C11-L186C23
+
+The `split_enums` `RecordSet` is declared as a `name` (on which join is done), and the semantic value: a `sc:URL` of type [`wd:Q3985153`](https://www.wikidata.org/wiki/Q3985153):
+
+https://github.com/mlcommons/datasets_format/blob/2e05069f02a93e6154137aa267e69bd1271a95b0/datasets/coco2014/metadata.json#L128-L153
+
+and the data describing those splits actually come from a [CSV file](https://github.com/mlcommons/datasets_format/blob/main/datasets/coco2014/data/splits.csv):
+
+https://github.com/mlcommons/datasets_format/blob/83dd4919ea32319703a0fa23ed013719548b4c03/datasets/coco2014/data/splits.csv#L1
+
+As you can see, here the split URLs are from the [known supported types](#known-supported-data-types) (under `wd:Q3985153`):
+ - https://mlcommons.org/definitions/training_split
```

Comment:
These pages do not exist yet. Should we add a "under construction" remark?

Reply:
Absolutely, added a warning.